### PR TITLE
Direct link to Tampermonkey chrome extension.

### PR DESCRIPTION
**Pull request by rodrigograca31**
Direct link to Tampermonkey chrome extension.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 [![Starve.io Extensions Reddit](http://i.imgur.com/ccMEPJK.png "Starve.io Extensions Reddit")](https://www.reddit.com/r/starveio/comments/6xiaec/starveio_extensions_github_compliments_to/)
 
 # TO USE / INSTALL
-1. Install [Tampermonkey](https://tampermonkey.net)
+1. Install [Tampermonkey](https://chrome.google.com/webstore/detail/tampermonkey/dhdgffkkebhmkfjojejmpbldmpobfkfo)
```
Comment (jasonkhanlar) [inline README.md:7]: Tampermonkey is compatible with Mozilla Firefox, Opera Next, Dolphin Browser, UC Browser, Google Chrome, Chromium, etc.

**Comment by jasonkhanlar** [comment]
Tampermonkey is compatible with Mozilla Firefox, Opera Next, Dolphin Browser, UC Browser, Google Chrome, Chromium, etc.

**Comment by rodrigograca31** [comment]
Ok.... I admit I'm biased .... :smile: 
But 90% of people will be using Google Chrome.... :smiley: 
I just wanted to make it easy for everyone... because I had friends asking "should I install the Stable, Beta or Legacy?" ....

**Comment by jasonkhanlar** [comment]
commit 563ec1bfa3f39e465548fa78d11a9b5e5edb2965 should resolve this

**Comment by rodrigograca31** [comment]
Oh. YAY! :smile:

**Pull request closed without merging**